meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -373,6 +373,9 @@ OSTREE_BOOTLOADER:imx8mm-lpddr4-evk-ebbr ?= "grub"
 IMAGE_EFI_BOOT_FILES:sota:imx8mm-lpddr4-evk-ebbr ?= "${@make_efi_dtb_boot_files(d)}"
 WKS_FILE:sota:imx8mm-lpddr4-evk-ebbr ?= "efidisk-sota.wks.in"
 # iMX8MM EVKB revision
+KERNEL_MODULE_AUTOLOAD:imx8mm-lpddr4-evk-ebbr += "moal"
+KERNEL_MODULE_PROBECONF:imx8mm-lpddr4-evk-ebbr += "moal"
```

Comment:
Is there a reason to add this driver only for the ebbr variation of the board?

Reply:
Right, it shouldn't be ebbr specific.

`VARIABLE:board += ""`

IIRC this will also potentially cause issues with the assignment (var ?=, var:board = ""), which is why we usually just append here instead.

Reply:
> Is there a reason to add this driver only for the ebbr variation of the board?

This replace the previous `MACHINE_FEATURES:append:imx8mm-lpddr4-evk = " mxm-mwifiex-load"` that uses the `imx8mm-lpddr4-evk` machine override, a few lines below there is another but now for `imx8mp-lpddr4-evk`.

Reply:
> IIRC this will also potentially cause issues with the assignment (var ?=, var:board = ""), which is why we usually just append here instead.

Changed to use the append

Reply:
> > Is there a reason to add this driver only for the ebbr variation of the board?
> 
> This replace the previous `MACHINE_FEATURES:append:imx8mm-lpddr4-evk = " mxm-mwifiex-load"` that uses the `imx8mm-lpddr4-evk` machine override, a few lines below there is another but now for `imx8mp-lpddr4-evk`.

misunderstood this part and ebbr variation is my fault! now fixed